src/_includes/declarative.md

```diff
@@ -0,0 +1,66 @@
+## An introduction to declarative UI
+
+_If you are familiar with a react-style declarative framework
+such as React.js, you can skip this section. This section
+describes the conceptual difference between the declaractive
+style used by Flutter, and the imperative style used by many other
+UI frameworks._
+
+### Why a declarative UI? 
+
+Frameworks from Win32 to web to Android and iOS typically use
+an [imperative style](https://en.wikipedia.org/wiki/Imperative_programming)
```

Comment:
I wouldn't link anything here. The wiki page's domain is broader, encompassing the entire programming language rather than UI programming. In that domain, Dart is still an imperative language. So it would invite arguments.